backend/file-system/src/main/java/com/htec/filesystem/service/ShelfService.java

```diff
@@ -70,12 +70,12 @@ public boolean createShelf(CreateShelfRequestModel createShelfRequestModel, Long
     }
 
     @Transactional
-    public void updateIsDeletedShelf(AuthUser user, List<Long> shelfIds, boolean delete) {
+    public void updateDeletedShelf(AuthUser user, List<Long> shelfIds, boolean delete) {
```

Comment:
again, soft deleting multiple shelfs not just one

Reply:
This method will probably be deleted since we don't need soft delete on shelf